### PR TITLE
Update train_dreambooth.py: added option to shuffle class images

### PR DESCRIPTION
**Pull request by martianunlimited**
Added option to shuffle class images, which I found to be useful when training multiple concepts that belongs in the same class, in improving the fine-tuned model. Current implementation "reuses" the first num_class_images in the directory resulting in the same regularization images used multiple times and the training "overfitting" to the regularization examples. Updated code shuffles the list so that the odds of the same image used for regularization is lower. 
Note that this option only makes sense if the number of images in the class_data_dir is (much) larger than num_class_images, and this option DOES NOT change the functionality of block ln470-ln513 to generate extra class images.

### CODE REVIEW

**Comment by ShivamShrirao** [comment]
Hey, I actually have a bigger change planned which makes a new sampler for the dataset. That will also be doing this. Just haven't got time to complete it yet.

**Comment by martianunlimited** [comment]
Cool looking forward to the new sampler.
